implementations/rust/ockam/ockam_api/src/nodes/service/secure_channel.rs

```diff
@@ -166,12 +167,11 @@ impl NodeManager {
         let identity = if let Some(identity) = identity_name {
```

Comment:
Should we warn users if `vault` is set but not `identity`? Because then the `vault` argument is silently not used.

Reply:
Yes, that makes total sense. We can enforce that at the command layer as well, so the user can't run the command passing only the `vault` argument.